valhalla/jawn/src/managers/prompt/PromptManager.ts

```diff
@@ -45,6 +45,7 @@ export class PromptManager extends BaseManager {
       },
       prompt: requestResult.data.request_body,
       userDefinedId: `prompt-from-request-${requestId}-${Date.now()}`,
+      createdAt: requestResult.data.request_created_at ?? undefined,
```

Comment:
added this to prevent this from showing up:

![image](https://github.com/user-attachments/assets/65aa9307-4011-458b-929b-a553f00c75c9)